### PR TITLE
add more check on destination id, address and port for connectivity c…

### PR DESCRIPTION
**Pull request by junyezhu**
…heck cmdlet

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherConnectivityCommand.cs [hunk 1]
```diff
@@ -113,6 +113,16 @@ public override void Execute()
 
             MNM.ConnectivityParameters parameters = new MNM.ConnectivityParameters();
 
+            if (string.IsNullOrEmpty(this.DestinationId) && string.IsNullOrEmpty(this.DestinationAddress))
+            {
+                throw new ArgumentException("Connectivity check missing destination resource id or address");
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherConnectivityCommand.cs]: Please put these customer facing strings into the resource strings for this project

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherConnectivityCommand.cs [hunk 2]
```diff
@@ -113,6 +113,16 @@ public override void Execute()
 
             MNM.ConnectivityParameters parameters = new MNM.ConnectivityParameters();
 
+            if (string.IsNullOrEmpty(this.DestinationId) && string.IsNullOrEmpty(this.DestinationAddress))
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherConnectivityCommand.cs]: Please add a test (can be a unit test) that exercises the new code paths

**Comment by MikhailTryakhov** [comment]
live tests job is failed as well

**Comment by MikhailTryakhov** [comment]
why is it targeted to ```preview```???

**Comment by junyezhu** [comment]
@MikhailTryakhov Because this PR is a new feature, not relevant to the test fix.

**Comment by MikhailTryakhov** [comment]
@junyezhu  please taget all your PRs to monthly branch, not ```preview```

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherConnectivityCommand.cs (hunk 2 above)
Comment (junyezhu) [reply src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherConnectivityCommand.cs]: Already tested the new code path in Powershell. I don't think tests are necessary for these new change, it only throw exception on invalid input.

Comment (markcowl) [reply src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherConnectivityCommand.cs]: Then how do we prevent later changes from regressing this fix?

Comment (MikhailTryakhov) [reply src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherConnectivityCommand.cs]: we definetely need to cover all scenarios

Comment (markcowl) [reply src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherConnectivityCommand.cs]: Note: it is OK to file an issue to add test automation next release, just link the issue here and we can merge for this release

**Comment by maddieclayton** [comment]
@MikhailTryakhov Should this PR be merged into the october branch?

**Comment by MikhailTryakhov** [comment]
Hi, @maddieclayton 
no this is a release train  so it'll go directly to ```preview```

We've merged progress from ```preview``` so this branch contains too many changes comparing with ```network-october-release```

**Comment by MikhailTryakhov** [comment]
Made mistake, @maddieclayton  please merge to monthly, it's correct

**Pull request merged**